src/main/resources/templates/error.html

```diff
@@ -4,10 +4,12 @@
     <div class="col-md-12">
 
         <div class="alert alert-danger" role="alert">
-            <h4 class="alert-heading">Error {{status}} - {{error}}</h4>
+            <h4 class="alert-heading">{{status}}</h4>
             <p>{{message}}</p>
             <hr>
-            <p class="mb-0" style="white-space: pre-line;">{{trace}}</p>
+            <pre>
+{{trace}}
+            </pre>
```

Comment:
The formatting looks broken :sweat_smile:

Reply:
That's because of <pre> would insert the fiest line of the stack trace at column 16, and not on column.

But I guess this do not count, so I'll indent the trace :-).